src/EntityFramework/Infrastructure/LocalDbConnectionFactory.cs

```diff
@@ -25,6 +25,14 @@ public sealed class LocalDbConnectionFactory : IDbConnectionFactory
         private readonly string _baseConnectionString;
         private readonly string _localDbVersion;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalDbConnectionFactory"/> class.
+        /// </summary>
+        public LocalDbConnectionFactory()
+            : this("mssqllocaldb")
```

Comment:
Better hope they don't change it again!

Reply:
It's been the same for 4 major SQL Server versions now